proposals/lip-Introduce_PoA_module.md

```diff
@@ -0,0 +1,606 @@
+            "items": {
+                "type": "object",
+                "required": ["address", "weight"],
+                "properties": {
+                    "address": {
+                        "dataType": "bytes",
+                        "fieldNumber": 1
+                    },
+                    "weight": {
+                        "dataType": "uint64",
+                        "fieldNumber": 2
+                    }
+                }
+            }
+        },
+        "threshold": {
+            "dataType": "uint64",
+            "fieldNumber": 2
+        },
+        "validatorsUpdateNonce": {
+            "dataType": "uint32",
+            "fieldNumber": 3
+        },
+        "signature": {
+            "dataType": "bytes",
+            "fieldNumber": 4
+        },
+        "aggregationBits": {
+            "dataType": "bytes",
+            "fieldNumber": 5
+        }
+    },
+    "required": [
+        "newValidators",
+        "threshold",
+        "validatorsUpdateNonce",
+        "signature",
+        "aggregationBits"
+    ]
+}
+```
+
+##### Verification 
+
+Let `trs` be a transaction with module ID `MODULE_ID_POA` and command ID `COMMAND_ID_UPDATE_AUTHORITY` to be verified. 
+The list of verification conditions for `trs.params` is as follows:
+
+*   Rules for `trs.params.newValidators` array:
+    *   The array must have at least 1 element and at most `MAX_NUM_VALIDATORS` elements.
+    *   The array must be ordered lexicographically by `address` property.
+    *   The `address` property of each element is a unique 20-byte address.
+    *   For every element `validator` in the `trs.params.newValidators` array, there is an entry with `storeKey == validator.address` in the validator substore.  
+    *   The `weight` property of each element is a positive integer.
+    *   Let `totalWeight` be the sum of the `weight` property of every element in the `trs.params.newValidators` array. Then `totalWeight` has to be less than or equal to `MAX_UINT64`.
+
+*   Rules for `trs.params.threshold` property:
+    *   The value of `trs.params.threshold` is within the following range:
+        *   Minimum value: ⌊ ⅓ × `totalWeight`⌋+ 1
+        *   Maximum value: `totalWeight`
+
+       where ⌊⋅⌋ is the floor function.
+
+*   Rules for `trs.params.validatorsUpdateNonce` property:
+    *   The value of `trs.params.validatorsUpdateNonce` has to be equal to `chainProperties.validatorsUpdateNonce`.
+*   Rules for `trs.params.aggregationBits` and `trs.params.signature` properties:
+    *   The function `verifyWeightedAggSig(keyList, tag, netID, aggregationBits, signature, m, weights, threshold)`, specified in [LIP 0038][BLS], must return VALID where:
+        *   The `keyList` property is an array containing `getValidatorAccount(address).blsKey` for every `address` property in `snapshotStore(0).validators` array sorted in the same order, where `getValidatorAccount` is the function exposed by the [validators module][validatorsModule].  
```

Comment:
Comparing with the mainchain registration, maybe we also want to sort the keys and weights array lexicographically by BLS key here, see https://github.com/LiskHQ/lips/pull/109#discussion_r709184436.

Reply:
good point, added